parts/k8s/kubernetescustomscript.sh

```diff
@@ -71,7 +71,9 @@ fi
 
 installContainerRuntime
 installNetworkPlugin
-installContainerd
+if [[ "$CONTAINER_RUNTIME" == "clear-containers" ]] || [[ "$CONTAINER_RUNTIME" == "kata-containers" ]] || [[ "$CONTAINER_RUNTIME" == "containerd" ]]; then
```

Comment:
We only actually install the containerd artifacts for required scenarios at provisioning time.